loadtest/control/simulcontroller/controller.go

```diff
@@ -288,6 +292,30 @@ func getActionList(c *SimulController) []userAction {
 			frequency:        0.0001, // https://mattermost.atlassian.net/browse/MM-61131
 			minServerVersion: semver.MustParse("10.0.0"),
 		},
+		{
+			name:             "CreateScheduledPost",
+			run:              c.createScheduledPost,
+			frequency:        0.2,
+			minServerVersion: semver.MustParse("10.3.0"),
+		},
+		{
+			name:             "UpdateScheduledPost",
+			run:              c.updateScheduledPost,
+			frequency:        0.1,
+			minServerVersion: semver.MustParse("10.3.0"),
+		},
+		{
+			name:             "DeleteScheduledPost",
+			run:              c.deleteScheduledPost,
+			frequency:        0.1,
+			minServerVersion: semver.MustParse("10.3.0"),
+		},
+		{
+			name:             "SendScheduledPost",
+			run:              c.sendScheduledPostNow,
+			frequency:        0.1,
+			minServerVersion: semver.MustParse("10.3.0"),
+		},
```

Comment:
These frequencies seem to be quite lower in real servers. Following https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/coverage-frequency.md and taking a look at Community and the largest customer in Cloud, the ratio of these API handlers against CreatePost doesn't even show up in the time windows with the most traffic. The largest ratio over the last 7 days is 0.04, but that's on very low-traffic time windows.
In short: I'd apply a frequency of 0.001 (which is the one we use for minimally-used endpoints) to all of these actions.

Reply:
Done.